pkg/plugin/processor/builtin/avro_examples_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 //nolint:govet // a more descriptive example description
 func ExampleEncodeProcessor_autoRegister() {
-	url, cleanup := schemaregistry.ExampleSchemaRegistryURL("ExampleEncodeProcessor_autoRegister")
+	url, cleanup := schemaregistry.ExampleSchemaRegistryURL("ExampleEncodeProcessor_autoRegister", 54321)
```

Comment:
Are example tests ever run in parallel (asking because the same port is used for the HTTP example)?

Reply:
I don't think they are, even if we used the `-parallel` flag it would only run tests that call `t.Parallel()`, but since there's no such function in examples I'd expect that they are always run sequentially.